wp-content/themes/core/assets/js/admin/ready.js

```diff
@@ -49,6 +50,9 @@ const domReady = () => {
 	// initialize admin block animation settings
 	animation();
 
+	// initialize admin stacking order settings
+	stackingOrder();
```

Comment:
I wonder if we can look into a core/column block name space/storage location for this script. I don't like adding it as a "global" or other generally-located script file in the directory structure. (applicable to both the script and the pcss partial.)

Reply:
I've moved the code to the `column` directory under `blocks/core/`. This code should now be attached to the Column block, rather than attached globally. The name is currently `admin.js` for that file, but this will be updated to `editor.js` with the changes to enqueuing.